docs/development/system-test-suites.md

```diff
@@ -0,0 +1,15 @@
+# Current Slow and Expensive test suites
+### This looks imbalanced but EKSBulkImportST is a lot slower than others
+| Slow1                      | Slow2                      | Slow3                           |
+| -------------------------- | -------------------------- | ------------------------------- |
+| AutoStopEcsTaskST          | EksBulkImportST            | CompactionCreationST            |
+| AutoDeleteS3ObjectsST      | RedeployOptionalStacksST   | MultipleTablesST                |
+| CompactionOnEC2ST          |                            | StateStoreCommitterThroughputST |
+| EmrPersistentBulkImportST  |
+| OptionalFeaturesDisabledST |
```

Comment:
These tests all run in the REENABLE_OPTIONAL_STACKS instance:
- RedeployOptionalStacksST
- AutoStopEcsTaskST
- AutoDeleteS3ObjectsST

Those are currently in different test suites, which means that instance is created multiple times. It might save time and resources overall if we put those tests in the same suite? Is there a class in Slow1 that we could swap with RedeployOptionalStacksST?